projects/lib/src/uciengine.cpp

```diff
@@ -54,7 +54,7 @@ QString variantFromUci(QString str, bool uciPrefix = true)
 
 QString variantToUci(const QString& str, bool uciPrefix = true)
 {
-	if (str.isEmpty() || str == "standard")
+	if (str.isEmpty())
```

Comment:
if (str.isEmpty() || (str == "standard" && uciPrefix == true))  // to be safe

Reply:
Is it necessary though? I don't want to add more spaghetti to this code if it isn't needed. That is no fault of cutechess of course, chess variants in uci is a mess and cutechess is just doing its best keep up.

To be honest I don't see why this function should ever return an empty string at all, it just seems like begging for trouble.

Reply:
AFAIK it is needed. There is no option `UCI_Standard`. Omitting the second expression sends `setoption UCI_Standard value true` to either engines without `UCI_Variant` combobox option  like mainline _Stockfish_ and _Komodo_   or such engines like _Sjaak II_ with `UCI_Variant` option but with `var chess` instead of`var standard` in the list of values.

Reply:
Sorry for forgetting about this PR. I've amended the change like you suggested, and it looks like it is still compatible with latest master